components/clubList.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import React, { PureComponent } from 'react';
+import { FlatList, View } from 'react-native';
+import { List, ListItem, Text } from 'react-native-elements';
+import connect from '../redux/connect';
+import * as actions from '../redux/modules/clubs';
+import { getIsLoading, getClubs } from '../redux/selectors/clubs';
+import noop from '../utils/noop';
+
+
+@connect(
+  {
+    clubs: getClubs,
+    isLoading: getIsLoading
+  },
+  { load: actions.load }
+)
+class ClubList extends PureComponent {
+  static defaultProps = {
+    clubs: [],
+    isLoading: false,
+    load: noop
+  };
+
+  componentDidMount() {
+    this.props.load();
+  }
+
+  renderItem = ({ item }) => (
+    <ListItem
+      title={`${item.name}`}
+      rightTitle={`${item.points} Bodova`}
+      rightTitleStyle={{ color: '#ff5252' }}
```

Comment:
My vote goes to keeping `styles` at the bottom of the `component` file.
It's not a big deal, but components would have less information about **how** they're rendered which would make it easier to see **what** they render.